permission/src/main/java/org/wildfly/security/permission/IntNameSetPermissionCollection.java

```diff
@@ -94,6 +90,10 @@ public boolean hasMoreElements() {
             return bits != 0;
         }
 
+        private Permission permissionFor(int id) {
```

Comment:
@deepalik-neu Just a total minor, since this method is `private` , can you please move it under all the public methods? It is our code style to have private methods located under the public methods. Thank you!